src/secp256r1/curve.rs

```diff
@@ -75,77 +76,97 @@ new_curve_impl!(
     SECP_A,
     SECP_B,
     "secp256r1",
-    |_, _| unimplemented!(),
+    |curve_id, domain_prefix| simple_svdw_hash_to_curve(curve_id, domain_prefix, Secp256r1::SSVDW_Z),
 );
 
-#[test]
-fn test_curve() {
-    crate::tests::curve::curve_tests::<Secp256r1>();
+impl Secp256r1 {
+    // Optimal Z with: <https://datatracker.ietf.org/doc/html/rfc9380#sswu-z-code>
+    // 0xffffffff00000001000000000000000000000000fffffffffffffffffffffff5
+    const SSVDW_Z: Fp = Fp::from_raw([
+        0xfffffffffffffff5,
+        0x00000000ffffffff,
+        0x0000000000000000,
+        0xffffffff00000001,
+    ]);
```

Comment:
Checked same as the suite `Z = -10` (https://www.rfc-editor.org/rfc/rfc9380.html#section-8.2)